plugins/repository-s3/README.md

```diff
@@ -0,0 +1,21 @@
+# repository-s3
+
+The repository-s3 plugin enables the use of S3 as a place to store snapshots.
+
+## Testing
+
+### Unit Tests
+
+```
+./gradlew :plugins:repository-s3:test
+```
+
+### Integration Tests
```

Comment:
1. Update this to share that setting up these vars when using actual s3 end-point is desired. Use this when there are changes in s3-plugin.
2. Add another section when user doesn't care about using actual s3.
```
 If using actual s3 service is not desired, we can skip setting up these vars. The plugin defaults to dummy  s3 service (using fixtures) which does not need above environment variables. 

./gradlew :plugins:repository-s3:s3ThirdPartyTest
```

Reply:
So this is no longer working and is https://github.com/opensearch-project/OpenSearch/issues/7867. I've added a note there to add to the documentation when we fix it.